rust/src/bin/loadtest/worker.rs

```diff
@@ -0,0 +1,44 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+use std::time::Instant;
+
+use log::*;
+use scylla::Session;
+use uuid::Uuid;
+
+use crate::stats::Stats;
+
+use care_pet::duration::Duration;
+use care_pet::{insert_query, random_sensor_data, Measure, Sensor};
+
+pub async fn work(sess: &'static Session, id: usize, sensors: Arc<HashMap<Uuid, Vec<Sensor>>>) {
+    debug!("worker # {} ready", id);
+
+    let prefix = format!("#{}", id);
+    let mut s = Stats::new();
+
+    loop {
+        for (_, sensors) in sensors.iter() {
```

Comment:
HashMap also implements Values, which gives you an iterator over values and would simplify the code:
```rust
for sensors in sensors.values() {
```